examples/SinusoidInterface/calibrate_script

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# paths
+out_dir=output
+data_path=truth.jld2
+eki_path=eki.jld2
+toml_path=priors.toml
+
+
+# seed
+rng_seed=235487
+rng_seed2=87653
+
+# parameters
+N_ensemble=5
+N_iterations=5
```

Comment:
I would not have them equal, it is easy to get loops wrong this way (No trivial out-of-bounds errors).

Reply:
This is taken from the sinusoid example, I've updated to 6 members. As the results are not exactly comparable anyway as I use differe random seeds anyway